### ISSUE TITLE
Parsing error: Subsequent lines with self-closing JSX tags

### ISSUE THREAD

**Issue by cpboyd** [mdx-js/eslint-mdx#138]
### Subject of the issue

Two lines with self-closing JSX tags seem to have issues:
```
error  Parsing error: unknown jsx node: "<Hero />\n<Featured />"
```

### Your environment

*   **OS**: macOS 10.15.2
*   **Packages**:
```
    "@typescript-eslint/eslint-plugin": "^2.17.0",
    "@typescript-eslint/parser": "^2.17.0",
    "eslint": "^6.8.0",
    "eslint-config-prettier": "^6.9.0",
    "eslint-config-react-app": "^5.1.0",
    "eslint-plugin-flowtype": "^4.6.0",
    "eslint-plugin-graphql": "^3.1.1",
    "eslint-plugin-import": "^2.20.0",
    "eslint-plugin-jsx-a11y": "^6.2.3",
    "eslint-plugin-mdx": "^1.6.5",
    "eslint-plugin-prettier": "^3.1.2",
    "eslint-plugin-react": "^7.18.0",
    "eslint-plugin-react-hooks": "^2.3.0",
```
*   **Env**: yarn 1.21.1

### Steps to reproduce

```jsx
import Hero from '../components/hero';
import Featured from '../components/favorites';

<Hero />
<Featured />
```

### Expected behaviour

The parser should be able to handle two separate self-closing JSX tags on subsequent lines.

### Actual behaviour

The parser throws an error.

**Comment by JounQin** [comment]
Please provide a minimal runnable reproduction.

**Comment by cpboyd** [comment]
@JounQin Here you go: https://github.com/cpboyd/bug-mdx

**Comment by JounQin** [comment]
This error is thrown from https://github.com/mdx-js/eslint-mdx/blob/master/packages/eslint-mdx/src/traverse.ts#L57
and adjacent self-closing jsx nodes are not parsed separately, it can be handlered by wrapping with `<$></$>`, I'll try to fix it ASAP.

**Comment by JounQin** [comment]
@mdx-js/maintainers Should that be parsed as separated nodes via `remark-mdx`?

**Comment by JounQin** [comment]
@cpboyd You can use as following as workaround temporarily.

```html
<Hero /><Featured />
```

**Comment by JounQin** [comment]
@mdx-js/releasers Would you like to publish a patch version by `yarn publish`?

**Comment by JounQin** [comment]
@cpboyd Would you like to check whether v1.6.6 works?

**Comment by cpboyd** [comment]
@JounQin it works, thanks!
